airflow_clickhouse_plugin/hooks/clickhouse_hook.py

```diff
@@ -1,24 +1,27 @@
 from itertools import islice
 from typing import *
 
-from airflow.hooks.base_hook import BaseHook
+from airflow.hooks.dbapi import DbApiHook
 from clickhouse_driver import Client
 
 
-class ClickHouseHook(BaseHook):
-    DEFAULT_CONN_ID = 'clickhouse_default'
+class ClickHouseHook(DbApiHook):
+    conn_name_attr = 'clickhouse_conn_id'
+    default_conn_name = 'clickhouse_default'
 
     def __init__(
             self,
-            clickhouse_conn_id: str = DEFAULT_CONN_ID,
             database: Optional[str] = None,
+            *args,
+            **kwargs
```

Comment:
Please don't use flexible signature, it might lead to an unexpected behaviour. We are definitely sure this hook uses `clickhouse_conn_id` as an argument, so let's stick with that.

Reply:
Added it back